### ISSUE TITLE
pkg_resources version lookup fails for source not installed yet

### ISSUE THREAD

**Issue by ryanrichholt** [tgen/jetstream#85]
During development I've ran into errors with the way that version information is loaded in `jetstream/__init__.py`. In that module, `pkg_resources` is used to fetch version information, but it seems to only work if the package is installed, not when working directly on source. There are quite a few recommendations for how to handle version info, but most of them suffer from some rare problems like this. The goal is to keep the version info present in only a single file, but use it for setup and also package info.

**Comment by ryanrichholt** [comment]
Fixed by moving version into two places: `setup.py` and `jetstream/__init__.py` Made notes in dev documents about this. Implemented in #93 v1.6

**Comment by ryanrichholt** [comment]
Here is an example of the errors caused. Running tests with Github actions, the package is downloaded but not installed before running. When a `pkg_resources` function is used to fetch the version number, the command fails:

```
/opt/hostedtoolcache/Python/3.6.9/x64/lib/python3.6/site-packages/pkg_resources/__init__.py:784: DistributionNotFound
___________________________ TestCliModule.test_build ___________________________

self = <test.test_cli.TestCliModule testMethod=test_build>

    def test_build(self):
        """jetstream build should just render and build the template"""
        with open('testwf.jst', 'w') as fp:
            fp.write('- cmd: "true"\n  stdout: /dev/null\n')
    
        args = ['build', 'testwf.jst']
>       cli_main(args)

test/test_cli.py:221: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
jetstream/cli/__init__.py:227: in main
    parser = arg_parser()
jetstream/cli/__init__.py:196: in arg_parser
    version=pkg_resources.get_distribution('jetstream').version
/opt/hostedtoolcache/Python/3.6.9/x64/lib/python3.6/site-packages/pkg_resources/__init__.py:479: in get_distribution
    dist = get_provider(dist)
/opt/hostedtoolcache/Python/3.6.9/x64/lib/python3.6/site-packages/pkg_resources/__init__.py:355: in get_provider
    return working_set.find(moduleOrReq) or require(str(moduleOrReq))[0]
/opt/hostedtoolcache/Python/3.6.9/x64/lib/python3.6/site-packages/pkg_resources/__init__.py:898: in require
    needed = self.resolve(parse_requirements(requirements))
```
